ClemBot.Bot/bot/extensions.py

```diff
@@ -217,6 +234,22 @@ def claims_check(self, claims: t.Sequence[str | Claims]) -> bool:
         # check for intersection of two sets of claims, if there is one we have a valid user
         return len(set(str_claims).intersection(self.claims)) > 0
 
+    def docs_url(self) -> str | None:
+        """
+        Gets the full URL of the documentation for the command.
+        Uses the `docs_url` string stored in BotSecrets and appends the page and header, if given.
+
+        Returns:
+            The full URL to the documentation of the command or None if `BotSecrets.docs_url` is None or `page` is None.
+        """
+        if not secrets.docs_url or not self.page:
+            return None
+
+        return (
+            f"{secrets.docs_url}{'/' if not secrets.docs_url.endswith('/') else ''}"
+            f"{self.page}{f'#{self.header}' if self.header else ''}"
+        )
```

Comment:
Would `urljoin` work here https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urljoin

Reply:
Good suggestion! Just made this change.